bigbluebutton-html5/imports/ui/components/chat/notification/component.jsx

```diff
@@ -113,13 +113,22 @@ class ChatNotification extends Component {
       disableNotify,
       openChats,
       intl,
+      chatIsOpen,
+      currentChatID,
     } = this.props;
 
+    if (disableNotify) return;
+
+    const hasUnread = ({ unreadCounter }) => unreadCounter > 0;
+    const isNotNotified = ({ id, unreadCounter }) => unreadCounter !== this.state.notified[id];
+    const isPrivate = ({ id }) => id !== PUBLIC_KEY;
+    const chatClosed = ({ id }) => !chatIsOpen || id !== currentChatID;
```

Comment:
When no chat is open, `currentChatID` is undefined, thus you don't need of `!chatIsOpen`